app/src/app/api/v0/threads/route.ts

```diff
@@ -0,0 +1,28 @@
+import { db } from "@/models";
+import { apiHandler } from "@/util/api";
+import createHttpError from "http-errors";
+import { NextRequest, NextResponse } from "next/server";
+
+export const POST = apiHandler(async (req) => {
```

Comment:
can we move this to assistants/threads?
Threads is a word with a lot of meanings and I'd like it to be more specific

Reply:
moved it to \api\v0\assistants\threads\export